README.md

```diff
@@ -462,6 +462,10 @@ This repository contains a curated list of awesome open source libraries that wi
 * [Transformers](https://github.com/huggingface/transformers) ![](https://img.shields.io/github/stars/huggingface/transformers.svg?style=social) - Huggingface's library of state-of-the-art pretrained models for Natural Language Processing (NLP).
 
 
+## Industry Strength RecSys
```

Comment:
Does that fall into the computation optimization-related modules?

Reply:
I believe the "Optimized Computation" list contains general-purpose frameworks that can be used to speed up any Python code. NVIDIA Merlin is made exclusively for the recommendation system. It includes models for different stages of developing a recommendation system product. For example, for the retrieval stage of RecSys, one can select the Two Towers model, and for the ranking stage, one can use the DLRM model. Those models are based on different approaches (e.g. collaborative filtering) and make sense to use for building recommendation systems at scale. 

Reply:
@axsaucedo What do you think?

Reply:
Thanks for the initial comments @al-yakubovich @zhimin-z - I'll comment in line